### PR TITLE
Refactor logging for prepare and msg pkgs.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-617.

### CODE REVIEW

File: svc/dappvpn/main.go [hunk 1]
```diff
@@ -19,12 +19,14 @@ import (
 	"github.com/privatix/dappctrl/svc/dappvpn/msg"
 	"github.com/privatix/dappctrl/svc/dappvpn/prepare"
 	"github.com/privatix/dappctrl/util"
+	"github.com/privatix/dappctrl/util/log"
 )
```
Comment (furkhat) [inline svc/dappvpn/main.go]: I've seen source files where we name our log as `log2` and default log package as it is. Lets follow that convention in all files?

File: svc/dappvpn/msg/msg_test.go [hunk 2]
```diff
@@ -13,20 +13,23 @@ import (
 	"github.com/privatix/dappctrl/sesssrv"
 	"github.com/privatix/dappctrl/svc/dappvpn/mon"
 	"github.com/privatix/dappctrl/util"
+	"github.com/privatix/dappctrl/util/log"
```
Comment (furkhat) [inline svc/dappvpn/msg/msg_test.go]: I've seen source files where we name our log as `log2` and default log package as it is. Lets follow that convention in all files?

File: svc/dappvpn/main.go (hunk 1 above)
Comment (dzeckelev) [reply svc/dappvpn/main.go]: I deleted the default log package.

File: svc/dappvpn/msg/msg_test.go (hunk 2 above)
Comment (dzeckelev) [reply svc/dappvpn/msg/msg_test.go]: I deleted the default log package.

**Pull request merged**